examples/Viewer.tsx

```diff
@@ -524,9 +545,30 @@ class Viewer extends React.Component<{}, ViewerState> {
         }
     }
 
+    private downloadFile() {
+        const { simulariumFile } = this.state;
+        if (!simulariumFile) {
+            return;
+        }
+        let href = "";
+        if (!simulariumFile.data) {
+            href = `https://aics-simularium-data.s3.us-east-2.amazonaws.com/trajectory/${simulariumFile.name}`;
+        } else {
+            href = URL.createObjectURL(simulariumFile.data.getAsBlob());
+        }
+        const downloadLink = document.createElement("a");
+        downloadLink.download = simulariumFile.name;
+        downloadLink.style.display = "none";
+        downloadLink.href = href;
+        document.body.appendChild(downloadLink);
+        downloadLink.click();
+        document.body.removeChild(downloadLink);
+        URL.revokeObjectURL(downloadLink.href);
+    }
+
```

Comment:
Having this code in the viewer here will make the front end implementation much simpler. :)

Reply:
This is just the example viewer to test that it works. This doesn't get shipped with the viewer npm package